### PR TITLE
adding newrelic dotnet sdk [Public Preview]

### PR DESCRIPTION
**Pull request by vipray-ms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by vipray-ms** [comment]
Hey @KrzysztofCwalina , @tg-msft , @pallavit . We are already in Public preview and this PR is for .NET public preview version. It's a request if you can complete it as we are going for GA soon.

**Comment by jsquire** [comment]
@vipray-ms:  As this is a management plane library, please coordinate with @ArthurMa1978 for review and guidance.

File: sdk/newrelic/Azure.ResourceManager.Newrelic/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/newrelic/Azure.ResourceManager.Newrelic/CHANGELOG.md:3]: Please replace the `Unreleased` with the expected release date of this first beta

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/newrelic/Azure.ResourceManager.Newrelic/CHANGELOG.md (hunk 1 above)
Comment (vipray-ms) [reply sdk/newrelic/Azure.ResourceManager.Newrelic/CHANGELOG.md]: Done.

**Pull request merged**